File: Q9.cs
using System;

namespace Q9
{
    class Program
    {
        static void Main(string[] args)
        {
            ushort[] idade = new ushort[20];
            ushort soma = 0;
            byte vezes = 1, média;
            for (byte i = 0; i < idade.Length; i ++, vezes ++)
            {
                Console.Write("Digite a {0}ª idade: ", vezes);
                idade[i] = Convert.ToUInt16(ushort.Parse(Console.ReadLine()));
                soma = Convert.ToUInt16(soma + idade[i]);
            }
            média = Convert.ToByte(soma / 20);
            Console.WriteLine("A média das 20 idades que você digitou é {0} anos.", média);
        }
    }
}

File: Q9.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int idade[20], soma = 0, vezes = 1, media;
    for (int i = 0; i < sizeof idade / sizeof idade[0]; i ++, vezes ++)
    {
        printf("Digite a %dª idade: ", vezes);
        cin >> idade[i];
        soma = soma + idade[i];
    }
    media = soma / 20;
    printf("A média das 20 idades que você digitou é %d anos.\n", media);
    return 0;
}